### DISCUSSION TITLE
RBAC in SOC, disabling nodes/read privileges

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#8991] [General] [answered]
During a configuration of a custom user and role I've encountered a problem with hiding all nodes in grid page.
In practice I've created a test role with custom elastic privileges and than in custom_roles file I've assigned event-monitor role for my custom role and than I've removed permissions to view nodes for event-monitor role, but the user which uses mu custom role still see all nodes in grid page.

**Comment by jertel** [comment, marked as answer]
Node read access is partially implemented (API only). There's another piece of the project that needs adjusted to finish this. Keep an eye on our release notes in upcoming releases, as we'll note this this once it's completed.
